shell/creators/app/files/.gitlab-ci.yml

```diff
@@ -0,0 +1,14 @@
+image: registry.suse.com/bci/bci-base:latest
+
+stages:
+  - check_version
+  - build_catalog
+
+variables:
+  REGISTRY: $CI_REGISTRY
+  REGISTRY_USER: $CI_REGISTRY_USER
+  REGISTRY_PASSWORD: $CI_REGISTRY_PASSWORD
+  IMAGE_NAMESPACE: $CI_PROJECT_NAMESPACE/$CI_PROJECT_NAME
+
+include:
+  - remote: 'https://raw.githubusercontent.com/rancher/dashboard/master/.gitlab/workflows/build-extension-catalog.gitlab-ci.yml'
```

Comment:
Is gitlab regularly used in air-gap environments? Would this cause an issue?

Reply:
This could be an issue, would it make more sense to just make the pipeline contain the full jobs? That would make it more difficult to send updates to the pipeline but it would at least work in an airgapped gitlab instance.

Reply:
I guess we can just leave it until someone requests it.